spec/manageiq_performance/middleware_spec.rb

```diff
@@ -23,6 +23,16 @@ def self.test_profile_eval_script
       end
     EOF
   end
+
+  def self.test_profile_env_for
+    ManageIQPerformance.send :profile_env_for
+  end
+
+  def self.test_profile_env_for_in_block
+    [1].map do
+      ManageIQPerformance.send :profile_env_for
+    end.first
```

Comment:
lol.  I was thinking of using a block. but not much (any?) better

```ruby
-> { ManageIQPerformance.send :profile_env_for }.call
```

Reply:
Yeah, I wouldn't try and optimize this too much.  Basically just trying to get a uniq line number within a block call, as well as having the `profile_env_for` method be the return value for the wrapper method.

This isn't production code, so don't really care if it is a bit fugly.  I can explain the rational for it though if it is a bit confusing (which I admit, it is).